NYTPhotoViewer/Resource Loading/NSBundle+NYTPhotoViewer.m

```diff
@@ -12,6 +12,9 @@
 @implementation NSBundle (NYTPhotoViewer)
 
 + (instancetype)nyt_photoViewerResourceBundle {
+    #if SWIFT_PACKAGE
+    return SWIFTPM_MODULE_BUNDLE;
+    #endif
     static NSBundle *resourceBundle = nil;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
```

Comment:
Is the name used below still accurate given the change from `NYTPhotoView.bundle` to `Media.xcasssets` in the podspec?

Reply:
yes, because the contents of Media.xcassets are copied into `NYTPhotoViewer.bundle/Assets.car`

tested with BP:
po [UIImage imageNamed:@"NYTPhotoViewerCloseButtonX" inBundle:resourceBundle compatibleWithTraitCollection:nil]
<UIImage:0x600002a24c60 named(org.cocoapods.NYTPhotoViewer: NYTPhotoViewerCloseButtonX) {15, 15}>

tested visually:
![image](https://user-images.githubusercontent.com/325572/105075400-b7f6ee00-5a57-11eb-8308-a4a5e3e3eb8a.png)

Reply:
test with `actool`: 
```
$ strings  $B/Assets.car | grep -i closebutton 
NYTPhotoViewerCloseButtonX
NYTPhotoViewerCloseButtonXLandscape
NYTPhotoViewerCloseButtonXLandscape@2x.png
NYTPhotoViewerCloseButtonX@2x.png
```